conf/osp.yaml.template

```diff
@@ -1,16 +1,26 @@
 OSP:
   # Openstack to be added as a compute resource
-  # Openstack api url For eg: https://hostname:port/v3/auth/tokens
-  HOSTNAME: https://hostname:port/v3/auth/tokens
-  # Login for Openstack
-  USERNAME: psi-satellite-jenkins
-  # Password for Openstack
+  # hostname: Openstack api url For eg: https://hostname:port/v3/auth/tokens
+  HOSTNAME:
+    OSP16: http://hostname:5000/v3/auth/tokens
+    OSP17: http://hostname:5000/v3/auth/tokens
+  # username: Login for Openstack
+  USERNAME:
+  # password: Password for Openstack
   PASSWORD:
-  # Openstack tenant to be used
-  TENANT: satellite-jenkins
```

Comment:
We do provisioning testing of any provider using `wrapanapi` where we establish the connection to the provider using wrapanapi APIs for that provider and verify if the VM and its properties are actually provisioned.

By removing these settings would we be able to connect to OSP through wrapanapi from what other settings/parameters we have ?

Reply:
Yes, you are right @jyejare, I missed some settings. Updated now